vendor/github.com/openshift/library-go/pkg/operator/configobserver/node/observe_minimum_kubelet_version.go

```diff
@@ -0,0 +1,47 @@
+package node
+
+import (
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/klog/v2"
+)
+
+var minimumKubeletVersionConfigPath = "minimumKubeletVersion"
+
+// ObserveKubeletMinimumVersion watches the node configuration and generates the minimumKubeletVersion
+func ObserveMinimumKubeletVersion(genericListers configobserver.Listers, _ events.Recorder, existingConfig map[string]interface{}) (ret map[string]interface{}, errs []error) {
```

Comment:
This should be feature gated too (and will now need to be aware of the new authorization-mode in the carry patch).